index.html

```diff
@@ -112,7 +112,7 @@ <h3>Postdocs</h3> <!-- Ordered by last name, Research Scientists and Post Doctor
 <ul>
 <li><a href="https://sites.google.com/site/monsieurlelanc">Subhadeep Banik</a></li>
 <li>Marcel Busch</li>
-<li>Vero Estrada (Senior Research Scientist - DEDIS)</li>
+<li>Vero Estrada</li>
```

Comment:
Again, is Vero a post doc or a research scientist (added below)?

Reply:
Senior research scientist